src/Elasticsearch.Net/Domain/RequestParameters/FluentRequestParameters.cs

```diff
@@ -9,17 +9,21 @@ namespace Elasticsearch.Net
 	/// You can always pass a simple NameValueCollection if you want.
 	/// </summary>
 	/// <typeparam name="T"></typeparam>
-	public abstract class FluentRequestParameters<T> : IRequestParameters 
+	public abstract class FluentRequestParameters<T> : IRequestParameters
 		where T : FluentRequestParameters<T>
 	{
 		private IRequestParameters Self => this;
 
 		public abstract HttpMethod DefaultHttpMethod { get; }
 
+		#region Explicit implementation of IRequestParameters
```

Comment:
I wrapped this chunk by region to pay attention on explicit implementation of IRequestParameters and separate it from other fields:) If you dislike it I can remove this region.